thesheriff/infrastructure/controllers/gang_controller.py

```diff
@@ -103,16 +104,17 @@ def join_gang_endpoint(gang_id: int) -> Response:
 
         return jsonify(message)
 
-    @blueprint_gang.route('/gang', methods=['GET', 'POST'])
-    def create_gang_endpoint() -> Response:
-        if request.method == 'GET':
-            results = list_gangs.execute()
-            gangs = list()
-            for res in results:
-                gangs.append(dict({'id': res.id, 'name': res.name}))
-            message = {'message': 'Success', 'gangs': gangs}
-            return jsonify(message)
+    @blueprint_gang.route('/gang', methods=['GET'])
```

Comment:
or /gangs 🤔

Reply:
Personally, I like to think about it like:

* GET /gang -> all gangs
* GET /gang/<id> -> 1 gang

Tho the 2nd case is not included in the specs so we don't really needed.

Reply:
Great, lets merge it